.eslintrc.js

```diff
@@ -1,78 +1,38 @@
 module.exports = {
-  // So parent files don't get applied
   root: true,
-  env: {
-    es6: true,
-    browser: true,
-    node: true,
-    jest: true,
-  },
-  globals: {
-    expect: true,
-  },
   extends: 'airbnb',
   parser: 'babel-eslint',
   parserOptions: {
     ecmaVersion: 7,
     sourceType: 'module',
   },
-  plugins: [
-    'babel',
-    'jsx-a11y',
-    'mocha',
-  ],
+  env: {
+    jest: true,
+    browser: true,
+  },
   rules: {
-    'max-len': ['error', 100, 2, {
+    'max-len': ['error', 120, 2, {
```

Comment:
https://www.quora.com/Why-does-Google-Java-coding-style-guide-chose-maximum-of-100-characters-column-limit-per-line-why-not-120-just-curious-to-know